lms/templates/dashboard/_dashboard_course_listing.html

```diff
@@ -128,7 +116,19 @@ <h2 class="university">${get_course_about_section(course, 'university')}</h2>
             <span>${course.display_number_with_default | h} ${course.display_name_with_default}</span>
           % endif
         </h3>
-      </hgroup>
+        <p class="date-block">
+        % if course.has_ended():
+        ${_("Course Completed - {end_date}").format(end_date=course.end_datetime_text("DATE_TIME"))}
+        % elif course.has_started():
+        ${_("Course Started - {start_date}").format(start_date=course.start_datetime_text("DATE_TIME"))}
+        % elif course.start_date_is_still_default: # Course start date TBD
+        ${_("Course has not yet started")}
+        % else:   # hasn't started yet
+        ${_("Course Starts - {start_date}").format(start_date=course.start_datetime_text("DATE_TIME"))}
+        % endif
+        </p>
+        <p alt="university" class="university">${get_course_about_section(course, 'university')}</p>
+      </div>
 
```

Comment:
@talbs I suspect Design team will have an issue with this change (it puts the school name and the start date after the course title).  For screen reader users, it makes more sense to have them in this order, but it's not as visually appealing.  Do you have any other suggestions?  Can you apply any CSS magic here to keep this DOM order, but visually display it the old way?